libs/etterlatte-token-model/src/main/kotlin/no/nav/etterlatte/token/Bruker.kt

```diff
@@ -0,0 +1,49 @@
+package no.nav.etterlatte.token
+
+sealed class Bruker(
+    open val accessToken: String
+) {
+    abstract fun saksbehandlerIdentEllerSystemnavn(): String
+
+    abstract fun erMaskinTilMaskin(): Boolean
+
+    abstract fun saksbehandlerEnhet(saksbehandlere: Map<String, String>): String
+
+    companion object {
+        private fun erMaskinTilMaskin(oid: String?, sub: String?) = (oid == sub) && (oid != null)
+        fun of(accessToken: String, saksbehandler: String?, oid: String?, sub: String?): Bruker {
```

Comment:
kunne det gitt mening å opprettet brukere fra `TokenValidationContext` i stedet eller et tilsvarende objekt? Og heller gjort uthenting av oid, sub osv i den methoden her for å holde det på ett sted.